code/datums/helper_datums/getrev.dm

```diff
@@ -1,44 +1,40 @@
 /datum/getrev
-	var/originmastercommit
-	var/commit
-	var/list/testmerge = list()
+	var/commit  // git rev-parse HEAD
 	var/date
+	var/originmastercommit  // git rev-parse origin/master

```

Comment:
`origin/<current branch>` is more accurate, but w/e

Reply:
I suppose this is a difference in how TGS operates actually because tg shitrev code only tracks origin/master bleh

Reply:
it's possible in git to learn the remote + branch corresponding to a local branch (dmdoc does this) but it's kind of a PITA so I didn't want to include it for now